projects/ngneat/dialog/src/lib/dialog.service.ts

```diff
@@ -26,7 +26,19 @@ export class DialogService {
   private document = inject(DIALOG_DOCUMENT_REF);
   private globalConfig = inject(GLOBAL_DIALOG_CONFIG);
 
-  dialogs: DialogRef[] = [];
```

Comment:
This is a breaking change

Reply:
Correct. I can deprecate it and make an adapter to avoid the breaking change

Reply:
or you can just leave it as an array

Reply:
I think it makes more sense as a map no?